examples/whatbot.py

```diff
@@ -0,0 +1,19 @@
+from jayk.cli.module import JaykMeta
+import string
+
+def message_is_what(message):
+    filtered_message = "".join(c for c in message if c not in (string.punctuation+string.whitespace))
```

Comment:
this seems superfluous. Just use `strip()` and `lower()` and compare the entire message against "what"